advocacy_docs/migrating/oracle/factors_to_consider.mdx

```diff
@@ -68,6 +69,20 @@ Migrating a legacy database to new database technology results in the need to up
 
 Also, when legacy databases are running on older specialized hardware and operating systems, a database migration might also require migrating the underlying host systems to modern platforms and operating systems.  For on-premises migrations, this can result in the need to change your operations and maintenance procedures and tools to work with the new server platforms and operating systems. For migrations to the cloud, some of the operations and maintenance activities will be handled by the cloud service provider but others will still be your responsibility. Likely, those activities will require different tools and procedures from the ones you're currently using.
 
+## Sizing and configuration considerations
+
+"How big a server will I need?" 
+
+This question comes up often when considering Oracle-to-Postgres migrations. The shortest answer is, "It depends," which isn't a good answer. A better one is, "Start with a 1:1 correspondence (that is, same number of cores, RAM, and so on), and go from there."
+
+Why is that?
+
+Postgres and Oracle are very different under the hood, with different SQL planning and different storage and access patterns. Even the version control and transaction isolation are implemented in a different way. That makes it difficult to set up a simple formula that covers all possible permutations of Oracle configurations and architectures. All databases respond differently to the workload being run.
+
```

Comment:
There was an incomplete sentence here that I deleted. The elaboration didn't seem necessary. But feel free to add it back and complete it if you do it's needeed.

Reply:
Agreed, elaboration isn't needed. Happy with this content. Can you commit this?